compiler/codegen/OMRCodeGenPhaseFunctionTable.hpp

```diff
@@ -26,6 +26,7 @@
    TR::CodeGenPhase::performReserveCodeCachePhase,                                           //ReserveCodeCachePhase
    TR::CodeGenPhase::performLowerTreesPhase,                                                 //LowerTreesPhase
    TR::CodeGenPhase::performSetupForInstructionSelectionPhase,                               //SetupForInstructionSelectionPhase
+   TR::CodeGenPhase::performUncommonCallConstNodesPhase,                                     //UncommonCallConstNodesPhase
```

Comment:
I think you want to do this before `SetupForInstructonSelection` has been done, since you're creating new nodes and you want them to go through the setup process.